### DISCUSSION TITLE
CCPA For California residents.

### DISCUSSION THREAD

**Discussion by Momo90k** [Poing-Studios/godot-admob-editor#80] [Q&A]
Does this plugin support CCPA?

**Comment by gumaciel** [comment]
Hi @Momo90k , it doesn't but we will provide this.

Issue: https://github.com/Poing-Studios/godot-admob-editor/issues/81

**Reply by Momo90k** [reply]
Can you please give me an estimated date? I'm really in a rush to use your plugin.
Thank you for your efforts.

**Reply by gumaciel** [reply]
Maybe 2 months, sorry but it's only me currently working on this plugin and there are many things to do.

You can try to develop it by yourself here: 

https://developers.google.com/admob/android/privacy/ccpa
https://developers.google.com/admob/ios/privacy/ccpa

**Reply by Momo90k** [reply]
I don’t know Java.

**Reply by gumaciel** [reply]
Development done on #82 , but need to testing
